setup.sh

```diff
@@ -6,6 +6,16 @@ echo "This should not be used in production. See https://github.com/owdex/compos
 # cd to script location, just in case
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
+echo "Checking for prerequisites..."
+docker --version > /dev/null || (echo "Docker not installed!"; exit 1)
+docker compose --help > /dev/null || (echo "Docker compose not installed!"; exit 1)
```

Comment:
Docker compose should not actually be required, owdex development works just as well with 
* A remote solr/mongo cluster
* an alternative container engine like podman

Reply:
Fair enough